sklearn/metrics/pairwise.py

```diff
@@ -198,6 +225,15 @@ def euclidean_distances(X, Y=None, Y_norm_squared=None, squared=False,
         Pre-computed dot-products of vectors in X (e.g.,
         ``(X**2).sum(axis=1)``)
 
+    kill_missing : boolean, optional
+        Allow missing values (e.g., NaN)
+
+    missing_values : String, optional
```

Comment:
Why a string? Surely we only want this configurable in the case of integer data...?